### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jeremy-farrance**
Tried to do the Build on a new machine without Visual Studio installed yet. Doing things in VS Code and Powershell, the build threw errors, 2 download/installs later and it worked.
- .NET Framework 5.x runtime
- .NET Framework 4.6.2 developer pack
 
Might be nice if someone improves my wording a bit?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ git remote add upstream https://github.com/DNNCommunity/DNNDocs
 Unless you are part of the DNNDocs core team, you will only have read access (you can only **pull**) to this remote.
 
 ## Running the DNN Docs Project Locally
+
+### .NET Frameworks?
```
Comment (valadas) [inline README.md:38]:
```suggestion
### .NET Frameworks
```

I would just remove the question mark...

**Review by valadas — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ git remote add upstream https://github.com/DNNCommunity/DNNDocs
 Unless you are part of the DNNDocs core team, you will only have read access (you can only **pull**) to this remote.
 
 ## Running the DNN Docs Project Locally
+
+### .NET Frameworks
+You should insure that you already have [.NET Framework v5.0](https://dotnet.microsoft.com/download/dotnet) and the "Developer Pack" for [.NET Framework v4.6.2](https://dotnet.microsoft.com/download/dotnet-framework/net462) install on your machine before trying the build process.
```
Comment (david-poindexter) [inline README.md:39]:
```suggestion
You should ensure that you already have [.NET 5.0](https://dotnet.microsoft.com/download/dotnet) (used by the `build` project) and the "Developer Pack" for [.NET Framework v4.6.2](https://dotnet.microsoft.com/download/dotnet-framework/net462) (used by the custom `DocFx` plugins) installed on your machine before trying the build process.
```

File: README.md [hunk 3]
```diff
@@ -34,6 +34,10 @@ git remote add upstream https://github.com/DNNCommunity/DNNDocs
 Unless you are part of the DNNDocs core team, you will only have read access (you can only **pull**) to this remote.
 
 ## Running the DNN Docs Project Locally
+
+### .NET Frameworks
```
Comment (david-poindexter) [inline README.md:38]:
```suggestion
### .NET Framework Prerequisites
```

**Review by david-poindexter — CHANGES_REQUESTED** [review]
Great catches @jeremy-farrance - thanks! 🎉

**Review by david-poindexter — APPROVED** [review]

**Review by valadas — APPROVED** [review]

**Pull request merged**